nationalarchives/ds-wagtail
# UN-411: Rename the `insights` app to `articles` (AB)

Related ticket(s):
https://national-archives.atlassian.net/browse/UN-411

## About these changes

- Renames the 'insights' Django app to 'articles'.
- Renames all models to use "Article" naming instead of "Insight" (e.g. `InsightPage` -> `ArticlePage`).
- Moves all of the templates and static files to "articles" folders and renames the files themselves to use "article" instead of "insight".
- Renames `featured_insight` relationship fields on other models to `featured_article`, as well as changing "insight" to "article" in method/property names dotted throughout the project.
- Adds a custom management command to directly rename the insight model tables to reflect the name change, and also modify the Django's 'migration record' table, so that Django doesn't think it needs to recreate everything.

## How to check these changes

1. Pull down the PR and run the `fab pull-production-data` command (NOTE: This will completely wipe out your current database)
2. Run `fab sh` to access the web container.
3. Use `python manage.py createsuperuser` to create yourself a new user to log in with.
4. Exit the web container 
5. Delete your local docker instances and run `fab run` to rebuild everything and run the Django app locally.
6. Head to http://0.0.0.0:8000/admin/pages/55/ in your browser. After logging in, you should see a list of 'Article pages' which were previously 'Insight pages', and you should be able to edit them and do all of the other things you do with pages - and they should behave just like the Insights pages did.

## Before assigning to reviewer, please make sure you have
